package/src/renderer/Canvas.tsx

```diff
@@ -93,6 +72,10 @@ const defaultFontMgr = Skia.FontMgr.RefDefault();
 
 export const Canvas = forwardRef<SkiaView, CanvasProps>(
   ({ children, style, debug, mode, onTouch, fontMgr }, forwardedRef) => {
+    const canvasCtx = useValue<{ width: number; height: number }>({
+      width: -1,
+      height: -1,
+    });
```

Comment:
Here it will be -1 always on the first render but I'm not sure how we could deal with this.

Reply:
That is the only way to go I think.